syft/core/frameworks/torch/tensor.py

```diff
@@ -2263,7 +2252,6 @@ def share(self, *workers):
                 if not hasattr(self, "grad") or self.grad is None:
                     self.init_grad_()
             n_workers = len(workers)
-            x_enc = self._encode()
```

Comment:
hey @trask I think this change is actually a mistake. We are no longer encoding the tensor for sharing

Reply:
did the formatter make this change?

Reply:
I don't think so @alok to confirm. It should not delete entire lines/functions.